src/main/kotlin/com/autonomousapps/transform/StandardTransform.kt

```diff
@@ -103,38 +103,40 @@ internal class StandardTransform(
     val usageIter = usages.iterator()
     while (usageIter.hasNext()) {
       val usage = usageIter.next()
-      val decl = declarations.find { it.variant(supportedSourceSets) == usage.variant }
+      val declarationsForVariant = declarations.filter { it.variant(supportedSourceSets) == usage.variant }.toSet()
```

Comment:
there's a `filterToSet` extension function that I'd prefer to use to reduce intermediate object creation.

Reply:
Done.